flybywiresim/aircraft
# feat(a380x/fcu): add TRUE indication on FCU when TRUE/MAG pb is pushed

Fixes #9519

## Summary of Changes


* add logic for TRUE indicator to display when true north ref is selected on AFS CP
* partial port of heading display to MSFS Avionics Framework

## Screenshots (if necessary)


![image](https://github.com/user-attachments/assets/8b890db6-8244-4f9a-9b97-680536221db5)

## References


![image](https://github.com/user-attachments/assets/85706bd6-3640-4459-9b20-5e9658319c39)





## Additional context



Discord username (if different from GitHub): Heclak

## Testing instructions


1. Select TRUE and check that TRUE light lights up in the AFS CP
2. Select MAG after check is complete and check that TRUE light is extinguished in the AFS CP.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
